### PR TITLE
Remove unnecessary span elements in Web API Docs

### PR DESCRIPTION
**Pull request by mfuji09**
> What was wrong/why is this fix needed? (quick summary only)

There are some span elements which are unnecessary in various documents.

> MDN URL of the main page changed

https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry/get
https://developer.mozilla.org/en-US/docs/Web/API/Element/slot
https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath
https://developer.mozilla.org/en-US/docs/Web/API/Media_Capabilities_API/Using_the_Media_Capabilities_API
https://developer.mozilla.org/en-US/docs/Web/API/MediaList
https://developer.mozilla.org/en-US/docs/Web/API/MediaList/mediaText
https://developer.mozilla.org/en-US/docs/Web/API/URL/port
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/set
https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Compressed_texture_formats

### CODE REVIEW

File: files/en-us/web/api/media_capabilities_api/using_the_media_capabilities_api/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 ---
 <div>{{APIRef("Media Capabilities API")}}</div>
 
-<p><span class="seoSummary">The <a href="/en-US/docs/Web/API/Media_Capabilities_API">Media Capabilities API</a> provides several key features to help you better decide how to handle media, but also to determine how well media is being handled, in real time.</span> These features include:</p>
+<p><span class="seoSummary">The <a href="/en-US/docs/Web/API/Media_Capabilities_API">Media Capabilities API</a> provides several key features to help you better decide how to handle media, but also to determine how well media is being handled, in real time. These features include:</p>
```
Comment (wbamberg) [inline files/en-us/web/api/media_capabilities_api/using_the_media_capabilities_api/index.html:17]: Looks like this removed the closing tag but not the opening one? If we added a paragraph break after "in real time." we could remove the opening tag as well.

**Review by wbamberg — CHANGES_REQUESTED** [review]
It looks like an error crept into one of these?

File: files/en-us/web/api/media_capabilities_api/using_the_media_capabilities_api/index.html (hunk 1 above)
Comment (mfuji09) [reply files/en-us/web/api/media_capabilities_api/using_the_media_capabilities_api/index.html]: It's my mistake. It shouldn't be delete in this subject. I have correct it. Thank you!

**Review by wbamberg — APPROVED** [review]
Thanks!

**Pull request merged**